django_app/redbox_app/templates/chats.html

```diff
@@ -93,7 +93,7 @@ <h3 class="rb-doc-selector__heading govuk-fieldset__heading">Documents to use</h
           <label class="govuk-body-s govuk-!-font-weight-bold" for="llm-selector">Model</label>
           <select id="llm-selector" name="llm" class="govuk-select govuk-!-margin-top-1">
             {% for option in llm_options %}
-              <option value="{{ option.name }}" {% if option.selected %}selected{% endif %}>{{ option.name }}{% if option.default %} (default){% endif %}</option>
+              <option value="{{ option.id }}" {% if option.selected %}selected{% endif %}>{{ option.name }}{% if option.default %} (default){% endif %}</option>
```

Comment:
the `ChatLLMBackend` is not uniquely defined by the `name` so we need to use the id instead